coremltools/converters/mil/mil/ops/defs/conv.py

```diff
@@ -22,8 +22,7 @@
 @register_op(doc_str="")
 class conv(Operation):
     """
-    Perform convolution over input. Currently supports only 1-D and 2-D
-    convolution.
+    Perform convolution over input. Supports 1-D, 2-D, and 3-D convolution.
 
```

Comment:
Need to update the line below as well: 
`* ``1 <= len(d_in) <= 2``: Only 1-D and 2-D convolution.` ---> `* ``1 <= len(d_in) <= 3`

Reply:
OK